src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/Tokenizer.java

```diff
@@ -1,5 +1,8 @@
 package org.rstudio.studio.client.workbench.views.source.editors.text.ace;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+
 import com.google.gwt.core.client.JavaScriptObject;
 
 public class Tokenizer extends JavaScriptObject
```

Comment:
Only on an explicit reformat gesture (e.g. pressing the icon or `CMD + SHIFT + A`).